src/applications/disability-benefits/2346/containers/ConfirmationPage.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import Scroll from 'react-scroll';
+import { focusElement } from 'platform/utilities/ui';
+
+const scroller = Scroll.scroller;
+const scrollToTop = () => {
+  scroller.scrollTo('topScrollElement', {
+    duration: 500,
+    delay: 0,
+    smooth: true,
+  });
+};
+
+export class ConfirmationPage extends React.Component {
+  componentDidMount() {
+    focusElement('.schemaform-title > h1');
+    scrollToTop();
+  }
+
+  render() {
+    // const { submission, data } = this.props.form;
+    // const { response } = submission;
+    // const name = data.veteranFullName;
+
+    return (
+      <div>
+        {/* <h3 className="confirmation-page-title">Claim received</h3>
+        <p>
+          We usually process claims within <strong>a week</strong>.
+        </p>
+        <p>
+          We may contact you for more information or documents.
+          <br />
+          <i>Please print this page for your records.</i>
+        </p>
+        <div className="inset">
+          <h4>
+            Form 2346 Claim <span className="additional">(Form 2346)</span>
+          </h4>
+          <span>
+            for {name.first} {name.middle} {name.last} {name.suffix}
+          </span>
+
+          {response && (
+            <ul className="claim-list">
+              <li>
+                <strong>Date received</strong>
+                <br />
+                <span>{moment(response.timestamp).format('MMM D, YYYY')}</span>
+              </li>
+            </ul>
+          )}
+        </div> */}
+        <h2>Your order is confirmed</h2>
+        <i className="form2346 fas fa-check-circle fa-4x" />
```

Comment:
This icon would look better if it was inside the `h2`. Right now it looks like some random check mark on the page. But I guess it's up to the design.

Reply:
done

Reply:
The check mark is too big!

![Screen Shot 2020-02-12 at 3 56 11 PM](https://user-images.githubusercontent.com/136959/74381115-7de58c80-4db0-11ea-969d-bd782a1d8388.png)

and I was thinking it should go on the left side of the text; but again, up to the design team.